lib/tasks/bulkupload.rake

```diff
@@ -0,0 +1,89 @@
+namespace :bulkupload do
+  desc "TODO"
+  task local: :environment do
+    require "find"
+
+    # MAKE THIS LOOK UP ktr_videoresources
+    contributer = User.find_by username: "ktr_videoresources"
+    videodirs = ["/home/jake/Videos/nzsl-share-1/", "/home/jake/Videos/nzsl-share-2/"]
```

Comment:
```suggestion
    videodirs = ENV.fetch('BULK_UPLOAD_PATHS', '').split(',')
```

I think it'd be good if we could at least pull these paths in from a basic env variable, to avoid hardcoding.

Reply:
fantastic suggestion